index.md

```diff
@@ -51,9 +51,9 @@ $ tavern-ci --stdout minimal_test.tavern.yaml
 
 ## Why not Postman, Insomnia or pyresttest etc?
 
-Tavern is a focused tool which does one thing well: automated testing of RESTful APIs.
+Tavern is a focused tool which does one thing well: automated testing of APIs.
 
-**Postman** and **Insomnia** are excellent tools which cover a wide range of use-cases, and indeed we use Tavern alongside Postman. However, specifically with regards to automated testing, Tavern has several advantages over Postman:
+**Postman** and **Insomnia** are excellent tools which cover a wide range of use-cases for RESTful APIs, and indeed we use Tavern alongside Postman. However, specifically with regards to automated testing, Tavern has several advantages over Postman:
 - A full-featured Python environment for writing custom validation functions
```

Comment:
I suggest mentioning MQTT as the second point for why Tavern is better than Postman. It's a big deal, don't shy away from it!